integration-tests/generate_from_templates.py

```diff
@@ -0,0 +1,108 @@
+parser.add_argument('--open-blobs-soft-limit', dest='open_blobs_soft_limit', type=int, default=2, help='soft limit for count of max blobs to remain in ram.')
+parser.add_argument('--open-blobs-hard-limit', dest='open_blobs_hard_limit', type=int, default=10, help='hard limit for count of max blobs to remain in ram.')
+parser.add_argument('-l', dest='bloom_filter_memory_limit', type=str, default='8 GiB', help='memory limit for all bloom filters. Unlimited if not specified.')
+parser.add_argument('-u', dest='auth_type', type=str, default='None', choices=['Basic', 'None'], help='auth type for bob')
+parser.add_argument('--index-memory-limit', dest='index_memory_limit', type=str, default='8 GiB', help='memory limit for all indexes')
+parser.add_argument('--enable-aio', dest='enable_aio', type=str, default='true', choices=['true', 'false'], help='enables linux AIO.')
+parser.add_argument('-p', dest='disks_events_logfile', type=str, default='/bob/log/bob_events.csv', help='path to logfile with info about disks states switches.')
+parser.add_argument('-b', dest='max_blob_size', type=str, default='100mb')
+parser.add_argument('--allow-duplicates', dest='allow_duplicates', type=str, default='true', choices=['true', 'false'], help='disables search for existing keys before write.')
+parser.add_argument('-d', dest='max_data_in_blob', type=int, default=10000)
+parser.add_argument('--blob-file-name-prefix', dest='blob_file_name_prefix', type=str, default='bob')
+parser.add_argument('--fail-retry-timeout', dest='fail_retry_timeout', type=str, default='100ms', help='retry to reinit pearl backend after fail.')
+parser.add_argument('--alien-disk', dest='alien_disk', type=str, default='d1')
+parser.add_argument('--bloom-filter-max-buf-bits-count', dest='bloom_filter_max_buf_bits_count', type=int, default=10000, 
+help='sets bloom filter buffer size in bits count, best value ~= max_data_in_blob.')
+parser.add_argument('--root-dir-name', dest='root_dir_name', type=str, default='bob')
+parser.add_argument('--alien-root-dir-name', dest='alien_root_dir_name', type=str, default='/bob/data/d1/alien')
+parser.add_argument('-t', dest='timestamp_period', type=str, default='1m', help='period when new pearl directory created.')
+parser.add_argument('--create-pearl-wait-delay', dest='create_pearl_wait_delay', type=str, default='100ms', help='each thread will wait this period if another thread creating pearl.')
+parser.add_argument('--metrics-name', dest='metrics_name', type=str, default='bob', help='add base name for metrics.')
+parser.add_argument('--graphite-enabled', dest='graphite_enabled', type=str, default='true', choices=['true', 'false'])
+parser.add_argument('--prometheus-enabled', dest='prometheus_enabled', type=str, default='false', choices=['true', 'false'])
+parser.add_argument('--path', dest='path', type=str, help='sets path to directory where configs will be generated.', default='/tmp')
```

Comment:
Too many arguments. IMHO it is better to get optional arguments from environment variables

Reply:
It seems that getting some variables as enviromental and others as script arguments would make code messier at render step. At this point, rendering variables for templates is done with one line: 
`f.write(template.render(vars(args), node_number=item))` 
and taking them from enviromental variables would demand explicitly equating them to their template counterpart, i.e: 
`f.write(template.render(log_config=${BOB_LOG_CONFIG}, user_config=${BOB_USER_CONFIG}, ...))`
Maybe it would be better left as it is?

Reply:
We decided to keep things as is